tbx/core/blocks.py

```diff
@@ -297,6 +298,42 @@ class IconChoice(models.TextChoices):
     WAGTAIL = "wagtail", "wagtail icon"
 
 
+class DivisionSignpostCardBlock(blocks.StructBlock):
+    class ColourTheme(models.TextChoices):
+        CORAL = "theme-coral", "Coral"
```

Comment:
Just to check: is this the new theme naming for the whole site? Replacing banana etc?

Reply:
This block has a reduced number of available colours compared to the rest of the site.

The theme colours that will be available to the site are: coral, nebuline, lagoon and green, but for this block, only coral, nebuline and lagoon have been designed (for charity, public sector and wagtail respectively), so I made it its own subclass of colour options.